File: C++/9thOct-2.cpp
#include<stdio.h>


int fact(int);

int main()

{
    
	int a,b,res=0;

    
    scanf("%d%d",&a,&b);

    
    if(a<b)

	       res=fact(a);

    
    else
 
	       res=fact(b);


    
    printf("%d ",res);

        return 0;

}

int fact(int x)

{
    
	if(x==0||x==1)
	
        return 1;

        else

	        return(x*fact(x-1));

}

File: Python/9thoct-2.py
def fact(n):
    if(n<1):
        return 1
    else:
        return n*fact(n-1)

def main():
    print("Enter number: ")
    inp=input()
    a,b=map(int,inp.split())
    res=fact(min(a,b))
    print(res)

main()